tests/filter/gpgsig.t

```diff
@@ -35,3 +35,25 @@ If 0b4cf6c9efbbda1eada39fa9c1d21d2525b027bb shows up then the signature was lost
   $ git rev-parse master double-filtered
   cb22ebb8e47b109f7add68b1043e561e0db09802
   cb22ebb8e47b109f7add68b1043e561e0db09802
+
+Remove the signature, the shas are different.
+  $ josh-filter :DROP_SIGNATURES refs/heads/master --update refs/heads/filtered -s
```

Comment:
Looks all good to me, except I'm not sure about the name `:DROP_SIGNATURES` maybe `:unsigned` (yes lower case)